src/common/rbac/namespace.go

```diff
@@ -31,31 +31,31 @@ type Namespace interface {
 }
 
 type projectNamespace struct {
-	projectIDOrName interface{}
-	isPublic        bool
+	projectID int64
+	isPublic  bool
 }
 
 func (ns *projectNamespace) Kind() string {
 	return "project"
 }
 
 func (ns *projectNamespace) Resource(subresources ...Resource) Resource {
-	return Resource(fmt.Sprintf("/project/%v", ns.projectIDOrName)).Subresource(subresources...)
+	return Resource(fmt.Sprintf("/project/%d", ns.projectID)).Subresource(subresources...)
 }
 
 func (ns *projectNamespace) Identity() interface{} {
-	return ns.projectIDOrName
+	return ns.projectID
```

Comment:
The function should return int64 rather than interface{}?

Reply:
Because the `Identity` of  `Namespace` interface returns `interface{}`.